server/services/timelineService.ts

```diff
@@ -69,6 +71,11 @@ export default class TimelineService {
     return Promise.all(eventPromises)
   }
 
-  private filterTimelineEvents = (event: TimelineEventResponse): boolean =>
-    SUPPORTED_TIMELINE_EVENTS.includes(event.eventType)
+  private filterTimelineEvents = (event: TimelineEventResponse): boolean => {
```

Comment:
I wasn't 100% happy with what I had. I originally wanted to combine two arrays, rather than having the "PLP" events listed twice in two separate arrays, as you have above (the first of which ends up being spread over several lines due to the ESLint rules!).

I originally tried the `push()` function, but it returns an integer (number of items added), rather than the array itself, however, I've just found the `concat()` function instead. What do you think of this?

```
// TODO RR-610 - remove feature toggle and revert back to one SUPPORTED_TIMELINE_EVENTS array
const PLP_TIMELINE_EVENTS = ['ACTION_PLAN_CREATED', 'INDUCTION_UPDATED', 'GOAL_UPDATED', 'GOAL_CREATED']
const PRISON_TIMELINE_EVENTS = ['PRISON_ADMISSION', 'PRISON_RELEASE', 'PRISON_TRANSFER']

const SUPPORTED_TIMELINE_EVENTS = config.featureToggles.includePrisonTimelineEventsEnabled
  ? PLP_TIMELINE_EVENTS.concat(PRISON_TIMELINE_EVENTS)
  : PLP_TIMELINE_EVENTS
```

Reply:
Unfortunately neither of the above work. I suspect these constants are initialised before `config.featureToggles.includePrisonTimelineEventsEnabled` is resolved?

Reply:
I've pushed a change which is hopefully easier to read?